com.fugro.roames.displaz.json

```diff
@@ -0,0 +1,54 @@
+{
+  "id": "com.fugro.roames.displaz",
+  "branch": "master",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.11",
+  "sdk": "org.kde.Sdk",
+  "command": "displaz",
+  "tags": [
+    "master"
+  ],
+  "finish-args": [
+    "--socket=x11",
+    "--socket=wayland",
+    "--device=dri"
+  ],
+  "modules": [
+    {
+      "name": "displaz",
+      "buildsystem": "simple",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/nigels-com/displaz.git",
+          "branch": "flatpak"
+        },
+        {
+          "type": "shell",
+          "commands": [
+            "mkdir -p thirdparty/external/downloads",
+            "sed -i '/#    include <GL\\/glu.h>/d' thirdparty/glew/GL/glew.h"
+          ]
+        },
+        {
+          "type": "file",
+          "url": "http://download.savannah.nongnu.org/releases/openexr/ilmbase-2.2.0.tar.gz",
+          "dest-filename": "thirdparty/external/downloads/ilmbase-2.2.0.tar.gz",
+          "sha256": "ecf815b60695555c1fbc73679e84c7c9902f4e8faa6e8000d2f905b8b86cedc7"
+        }
+      ],
+      "build-commands": [
+        "mkdir build_external",
+        "pushd build_external && cmake ../thirdparty/external && make && popd",
+        "mkdir build",
+        "pushd build && cmake -DCMAKE_INSTALL_PREFIX=/app .. && make install && popd",
+        "mkdir -p /app/share/icons/hicolor/256x256/apps/",
+        "mkdir -p /app/share/applications/",
+        "mkdir -p /app/share/metainfo/",
+        "cp src/resource/displaz_icon_256.png            /app/share/icons/hicolor/256x256/apps/com.fugro.roames.displaz.png",
```

Comment:
What about smaller icons?

Reply:
Added 16, 32, 48 and 128 pixels versions. Thanks.

Reply:
There is an [SVG](https://github.com/nigels-com/displaz/blob/e4760d8b9a316d591cd9ac392bd891162aee90bd/package/icon.svg) icon here, so you can resize it using `rsvg-convert`.

Example:
https://github.com/flathub/com.github.PintaProject.Pinta/blob/8e13392e8ad5ee23af7e76edd5fe08fac046767e/Pinta.yaml#L13-L19

Reply:
Or just install the SVG icon in `/app/usr/share/icons/hicolor/scalable/`.

Reply:
@bilelmoussaoui 

> Or just install the SVG icon in `/app/usr/share/icons/hicolor/scalable/`.

Both SVG and PNGs should be installed.
https://github.com/flathub/flathub/pull/293#issuecomment-372499013